src/vs/platform/workspace/common/workspace.ts

```diff
@@ -78,6 +82,13 @@ export interface IWorkspaceRevState {
 	zapRef?: string;
 }
 
+declare class Map<K, V> {
```

Comment:
Also the previously imported `LinkedMap` does not update when `set(` is called, it returns false if a value was already present.